bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookKeeperTest.java

```diff
@@ -717,8 +717,6 @@ public void testReadEntryReleaseByteBufs() throws Exception {
                 for (Enumeration<LedgerEntry> readEntries = lh.readEntries(0, numEntries - 1);
                     readEntries.hasMoreElements();) {
                     LedgerEntry entry = readEntries.nextElement();
-                    assertTrue(entry.data.getClass().getName(),
-                        entry.data.getClass().getName().contains("PooledNonRetainedSlicedByteBuf"));
```

Comment:
why are you removing this checks ? they are 'poor man' checks about the fact that we are pooling/non-pooling the ByteBuf. it is important to check that in v2 protocol bytebufs are "pooled"

Reply:
this check is not correct, and depends heavily on the implementation of netty. because the class can be SimpleLeakAwareByteBuf.

Reply:
@eolivelli : you can check this - https://builds.apache.org/job/bookkeeper-precommit-pullrequest/org.apache.bookkeeper$bookkeeper-server/335/testReport/junit/org.apache.bookkeeper.client/BookKeeperTest/testReadEntryReleaseByteBufs/

Reply:
@sijie  OK I am convinced.
It is a pity that we are not able to made a usable assertion :-(

the error is
java.lang.AssertionError: io.netty.buffer.SimpleLeakAwareByteBuf
	at org.apache.bookkeeper.client.BookKeeperTest.testReadEntryReleaseByteBufs(BookKeeperTest.java:742)